src/plugin/fieldViews/RichTextFieldView.ts

```diff
@@ -27,6 +27,50 @@ export const createRichTextField = ({
   nodeSpec,
 });
 
+type FlatRichTextOptions = RichTextOptions & {
+  nodeSpec?: Partial<Omit<NodeSpec, "content">>;
+};
+
+/**
+ * Create a rich text field that contains only text (no p tag). Lines
+ * are separated by line breaks (`<br>` tags).
+ */
+export const createFlatRichTextField = ({
+  createPlugins,
+  nodeSpec,
+}: FlatRichTextOptions): RichTextField =>
+  createRichTextField({
+    createPlugins: (schema) => {
+      const br = schema.nodes.hard_break;
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- there's no guarantee this is present in the Schema.
+      if (!br) {
+        throw new Error(
+          "[prosemirror-elements]: Attempted to add a FlatRichTextField, but there was no hard_break node in the schema. You must supply one to use this field."
+        );
+      }
+
+      const keymapping = {
+        Enter: (state: EditorState, dispatch?: (tr: Transaction) => void) => {
+          dispatch?.(
+            state.tr.replaceSelectionWith(br.create()).scrollIntoView()
+          );
+          return true;
+        },
+      };
+
+      const plugin = keymap(keymapping);
+      return [plugin, ...(createPlugins?.(schema) ?? [])];
+    },
+    nodeSpec: {
+      ...nodeSpec,
+      content: "(text|hard_break)*",
```

Comment:
Could this have issue if we ever wanted other kinds of content in a flat field, i.e. lists? Perhaps this never happens but I'm not sure if that is made explicit from the name i.e. the name should make clear it can only have text and breaks. Naming is difficult!

Also the fact you can specify a content property that will then be ignored could perhaps be a bit confusing.

Reply:
It's flat! No-one is allowed a list 😁  I dislike the name also, but can't think of anything else at present – suggestions welcome, and we can easily change it down the line if we can't think of anything ATM.

_> Also the fact you can specify a content property that will then be ignored could perhaps be a bit confusing._

The type for the `RichTextOptions`, `Partial<Omit<NodeSpec, "content">>`, removes the `content` type from the nodeSpec, so you shouldn't be able to specify a content property when calling `createFlatRichTextField` (for the reason you mention.)

Reply:
> removes the content type from the nodeSpec

Oh I missed that, looks good.

> we can easily change it down the line if we can't think of anything ATM.

Rationale on name makes sense, let's go with that 👍